src/jit/SimdArm32Inl.h

```diff
@@ -682,7 +688,7 @@ static void emitUnarySIMD(sljit_compiler* compiler, Instruction* instr)
         break;
     case ByteCode::F32X4DemoteF64X2ZeroOpcode:
         simdEmitOp(compiler, SimdOp::Type::vcvtftf | (0x1 << 8), dst, 0, args[0].arg);
-        simdEmitOp(compiler, SimdOp::Type::vcvtftf | (0x1 << 8) | (0x1 << 22), dst, 0, getHighRegister(args[0].arg));
```

Comment:
`getHighRegister` should be deleted from the source code

Reply:
There's a little problem. It's still used in an other file (**MemoryInl.h**) by `sljit_emit_simd_mov(...)` and `sljit_emit_simd_replicate(...)` functions that cannot be altered withing the Walrus project (hence they're parts of the SLJIT). What shall I do?